packages/airnode-starter/scripts/choose-integration.ts

```diff
@@ -0,0 +1,88 @@
+import { readdirSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import prompts, { PromptObject } from 'prompts';
+import { IntegrationInfo } from '../src';
+
+const createOption = (name: string) => ({
+  title: name,
+  value: name,
+});
+
+const questions: PromptObject[] = [
+  {
+    type: 'select',
+    name: 'integration',
+    message: 'Choose integration',
+    choices: readdirSync(join(__dirname, '../integrations'))
+      .filter((integration) => integration !== 'README.md')
+      .map(createOption),
+  },
+  {
+    type: 'select',
+    name: 'airnodeType',
+    message: 'Choose Airnode type',
+    choices: [createOption('local'), createOption('aws')],
+  },
+  // TODO: move AWS secrets to separate script
+  {
+    type: (_prev, values) => (values.airnodeType === 'aws' ? 'text' : null),
+    name: 'accessKeyId',
+    message: [
+      'Since you want to deploy on AWS, we need your credentials.',
+      '',
+      'See video how to create these: https://www.youtube.com/watch?v=KngM5bfpttA',
+      '',
+      'Specify access key id',
+    ].join('\n'),
+  },
+  {
+    type: (_prev, values) => (values.airnodeType === 'aws' ? 'text' : null),
+    name: 'secretKey',
+    message: 'Specify secret access key',
+  },
+  {
+    type: 'select',
+    name: 'network',
+    message: 'Choose target network',
+    choices: [createOption('rinkeby'), createOption('localhost')],
+  },
+  {
+    type: (_prev, values) => (values.network !== 'localhost' ? 'text' : null),
+    name: 'mnemonic',
+    message: [
+      'Since you chose testnet network, we need an account with testnet funds to connect to the blockchain.',
+      '',
+      'DO NOT USE YOUR REAL WALLET!!!',
```

Comment:
"Real wallet" sounds a bit weird to me. What is a "real wallet"? I would rather phrase like:

```suggestion
      'IMPORTANT: DO NOT ENTER A MNEMONIC LINKED WITH MAINNET ACCOUNTS',
```